classes/02class/exercises/c02-network03/rodrigovcesar/README.md

```diff
@@ -0,0 +1,73 @@
+# c02-network03
+
+## Commands Execution Output
+
+- Commands for creating a new VPC:
+```
+aws ec2 create-vpc --cidr-block 10.0.0.0/16 --instance-tenancy default 
+{
+    "Vpc": {
+        "CidrBlock": "10.0.0.0/16",
+        "DhcpOptionsId": "dopt-a57d56c2",
+        "State": "pending",
+        "VpcId": "vpc-03f503f58eb9f0512",
+        "OwnerId": "102825027349",
+        "InstanceTenancy": "default",
+        "Ipv6CidrBlockAssociationSet": [],
+        "CidrBlockAssociationSet": [
+            {
+                "AssociationId": "vpc-cidr-assoc-07059c9abbf5c0c54",
+                "CidrBlock": "10.0.0.0/16",
+                "CidrBlockState": {
+                    "State": "associated"
+                }
+            }
+        ],
+        "IsDefault": false
+    }
+}
+
+aws ec2 create-tags --tags Key=Name,Value=devopsacademy-vpc --resources vpc-03f503f58eb9f0512
+
+aws ec2 describe-vpcs --vpc-id vpc-03f503f58eb9f0512
+{
+    "Vpcs": [
+        {
+            "CidrBlock": "10.0.0.0/16",
+            "DhcpOptionsId": "dopt-a57d56c2",
+            "State": "available",
+            "VpcId": "vpc-03f503f58eb9f0512",
+            "OwnerId": "102825027349",
+            "InstanceTenancy": "default",
+            "CidrBlockAssociationSet": [
+                {
+                    "AssociationId": "vpc-cidr-assoc-07059c9abbf5c0c54",
+                    "CidrBlock": "10.0.0.0/16",
+                    "CidrBlockState": {
+                        "State": "associated"
+                    }
+                }
+            ],
+            "IsDefault": false,
+            "Tags": [
+                {
+                    "Key": "Name",
+                    "Value": "devopsacademy-vpc"
+                }
+            ]
+        }
+    ]
+}
+
+
+
+```
+
+- Any extra challenges faced?
+
+To create associate the name I needed a second command I found outside the AWS docs.
```

Comment:
You can find the "tag specification" here: https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc.html

The syntax is a bit annoying: `ResourceType=vpc` for the example (you can check all possible values in the documentation, under "JSON Syntax"